src/settingsView.script.ts

```diff
@@ -97,10 +98,34 @@ window.addEventListener('message', event => {
         actionsElement.appendChild(actionElement);
     }
   } else if (method === 'models') {
-    const { configs, showModelSelector } = params;
+    const { configs, showModelSelector } = params as ModelsMessage;
     const select = document.getElementById('models') as HTMLSelectElement;
+
+    function updateConfigErrors(config: ConfigEntry) {
+      const configErrors = document.getElementById('model-errors') as HTMLParagraphElement;
+      configErrors.textContent = '';
+      for (const error of config.errors) {
+        if (!error.message)
+          continue;
+
+        const errorParagraph = document.createElement('p');
+        errorParagraph.textContent = 'Unable to load ';
+        if (error.location) {
+          const errorLink = document.createElement('a');
+          errorLink.textContent = `${error.location.file.split('/').pop()}`;
+          errorLink.role = 'link';
+          errorLink.onclick = () => vscode.postMessage({ method: 'openFile', params: { location: error.location! } });
+          errorParagraph.appendChild(errorLink);
+        } else {
+          errorParagraph.textContent += ' config file';
```

Comment:
Let's link to the config file in this case. Otherwise, it's not really actionable for the user.